py3status/modules/rss_aggregator.py

```diff
@@ -1,24 +1,38 @@
 # -*- coding: utf-8 -*-
 """
-Display the unread feed items in your OwnCloud/NextCloud account.
+Display the unread feed items in your favorite RSS aggregator.
+
+For now, supported aggregators are:
+    * OwnCloud/NextCloud with News application
+    * Tiny Tiny RSS 1.6 or newer
 
 You can also decide to check only for specific feeds or folders of feeds. To use this
 feature, you have to first get the IDs of those feeds or folders. You can get those IDs
-by clicking on the desired feed or folder and watching the URL:
+by clicking on the desired feed or folder and watching the URL.
+
+For OwnCloud/NextCloud:
 ```
 https://yourcloudinstance.com/index.php/apps/news/#/items/feeds/FEED_ID
 https://yourcloudinstance.com/index.php/apps/news/#/items/folders/FOLDER_ID
 
 ```
+For Tiny Tiny RSS:
+
+```
+https://yourttrssinstance.com/index.php#f=FEED_ID&c=0
+https://yourttrssinstance.com/index.php#f=FOLDER_ID&c=1
+```
+
 If both feeds list and folders list are left empty, all unread feed items will be counted.
-Counted items don't need to match both feeds and folders lists. Items matching both a
-given feed ID and a given folder ID will be counted only once.
+You may use both feeds list and folders list, but given feeds shouldn't be included in
+given folders, else unread count number behavior is unpredictable. Same warning when
+aggregator allows subfolders: the folders list shouldn't include a folder and one of its
+subfolder.
```

Comment:
Isn't this something you could cope with?  Doesn't each feed have a uuid that you can add to a set and then easily get the unique count?

Anyhow I haven't looked at the data that we get so this is just a guess.